apps/web/modules/settings/my-account/general-view.tsx

```diff
@@ -113,6 +114,12 @@ const GeneralView = ({ localeProp, user, travelSchedules, revalidatePage }: Gene
       if (res.locale) {
         window.calNewLocale = res.locale;
         document.cookie = `calNewLocale=${res.locale}; path=/`;
+        const newLocaleI18nTranslations = await loadTranslations(res.locale, "common");
+        window.APP_ROUTER_I18N = {
+          translations: newLocaleI18nTranslations,
+          locale: res.locale,
+          ns: "common",
+        };
```

Comment:
updating window.APP_ROUTER_I18N with new translations once user changes its language setting in /settings/my-account/general page